app/Models/Beatmapset.php

```diff
@@ -590,6 +591,12 @@ public static function searchSortParamsES(array $params)
             'title' => 'title.raw',
         ];
 
+        // subsorts
+        $subsort = [];
+        if ($params['sort_field'] === 'hype') {
```

Comment:
directly adding to `$sortFields` is probably better if using `if` for comparison.